### ISSUE TITLE
Kimai 1.30.4 does not run on PHP 8.0 

### ISSUE THREAD

**Issue by Sarmingsteiner** [kimai/kimai#3781]
### Describe the issue

It seems that Kimai 1.30.4 requires PHP 8.1 as minimum, while the release notes say 8.0.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

1.30.4

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.0

**Comment by skipperTux** [comment]
This is what I get running Composer update all dependencies:

```
[snip]
"Executing script cache:clear [KO]", " [KO]", "Script cache:clear returned with error code 255", "!!  ", "!!  Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version \">= 8.1.0\". Y
ou are running 8.0.26. in /www/htdocs/kimai.example.org/kimai2/vendor/composer/platform_check.php on line 24", "!!  PHP Fatal error:  Composer detected issues in your platform: Your Composer dependencies require a PHP version \">= 
8.1.0\". You are running 8.0.26. in /www/htdocs/kimai.example.org/kimai2/vendor/composer/platform_check.php on line 24", "!!  ", "Script @auto-scripts was called via post-install-cmd"]}
```

**Comment by kevinpapst** [comment]
Thanks for posting 👍  I shall not tag release in the middle of the night.
It was so bad, I even had to delete the tag, which I never did before... 🙈

Please upgrade to 1.30.5: https://github.com/kimai/kimai/releases/tag/1.30.5

**Comment by skipperTux** [comment]
And I should not update in the morning without testing before in my dev environment 🙈. Good thing restoring Kimai from a backup is a snap 😃. Thanks for the great work!
And thanks for the fast fix, no issues with the update to 1.30.5 and PHP 8.0.

Btw, running PHP 8.1 on all-inkl does not work, I guess it is related to #2859. Put it on my agenda to further investigation.

**Comment by kevinpapst** [comment]
I don't think a backup restore is even necessary, or was it?
You should be good with a regular update.

Yes, all-inkl with the ioncubeLoader is a massive pain... if you find something, I'd be glad to get help with that.
